app/controllers/admin/plugins_controller.rb

```diff
@@ -3,9 +3,18 @@
 class Admin::PluginsController < Admin::StaffController
   def index
     render_serialized(
-      Discourse.visible_plugins.sort_by { |p| p.name.downcase.delete_prefix("discourse-") },
+      Discourse.plugins_sorted_by_name(enabled_only: false),
       AdminPluginSerializer,
       root: "plugins",
     )
   end
+
+  private
+
+  def preload_additional_json
+    store_preloaded(
+      "enabledPluginAdminRoutes",
+      MultiJson.dump(Discourse.plugins_sorted_by_name.map(&:admin_route).compact),
```

Comment:
Can also be a single call:

```suggestion
      MultiJson.dump(Discourse.plugins_sorted_by_name.filter_map(&:admin_route)),
```